ffi_common/build.rs

```diff
@@ -1,17 +1,2 @@
-use cbindgen::{Builder, Language};
-use std::env;
 
-fn main() {
-    let crate_dir = env::var("OUT_DIR").unwrap();
-
-    Builder::new()
-        .with_crate(crate_dir)
-        .with_language(Language::C)
-        .with_parse_expand(&["ffi_common"])
-        .generate()
-        .map(|bindings| bindings.write_to_file("bindings.h"))
-        .unwrap_or_else(|_| {
-            eprintln!("Unable to generate bindings");
-            false
-        });
-}
+fn main() { }
```

Comment:
I forget--do you still need this for anything?

Reply:
Nope! Had to take another look, but only `ffi_derive` and `ffi_internals` need a `build.rs`, so this can go away. The output directory is passed in everywhere else.

Also gonna drop the `cbindgen` build dependency -- the docs already have notes on how to run that from the consumer crate, where it can create a header for the whole thing. It just stuck around from earlier versions.